integration-tests/metrics_test.go

```diff
@@ -173,9 +176,9 @@ func testMetrics(t *testing.T, metricsHandler *httpzdmproxy.HandlerWithFallback)
 			// 3 on async: AUTH_RESPONSE, STARTUP AND QUERY SELECT
 			// ONLY QUERY is tracked
 			if conf.ReadMode == config.ReadModeDualAsyncOnSecondary {
-				time.Sleep(1000 * time.Millisecond)
+				time.Sleep(10000 * time.Millisecond)
```

Comment:
10000 is a bit too much... What failure were you seeing with 1000 before?

Reply:
That must have been a leftover from some experiments I did, but I ran successfully with the original value now on local. Just reverted that.